packages/cadl-ranch-specs/http/versioning/madeOptional/main.tsp

```diff
@@ -0,0 +1,64 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@typespec/versioning";
+
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+/**
+ * Test for the `@madeOptional` decorator.
+ */
+@service
+@versioned(Versions)
+@server(
+  "{endpoint}/versioning/made-optional/api-version:{version}",
+  "Testserver endpoint",
+  {
+    /**
+     * Need to be set as 'http://localhost:3000' in client.
+     */
+    endpoint: url,
+
+    /**
+     * Need to be set as 'v1' or 'v2' in client.
```

Comment:
Any of the test require we set `v1` in client?

Reply:
nope. i just want to add test for latest signature after versioning projection for now.